web/index.html

```diff
@@ -85,113 +84,166 @@ <h2 style="margin: 0;">Enjoying my site?</h2>
             </div>
             <transition name="slide-up" mode="out-in">
                 <div key=0 v-if="currentElement">
+                    <button class="controls left-arrow" :style="`top:max(${scroll}px, 300px);`" @click="slideLeft"></button>
```

Comment:
Did we remove thses?

Reply:
the svgs, yes, the buttons that allow you to switch between panels, no. That's how you switch between panels.